src/test/java/org/craftercms/studio/impl/v2/repository/blob/BlobAwareContentRepositoryTest.java

```diff
@@ -25,7 +25,7 @@
 import org.craftercms.studio.api.v1.to.VersionTO;
 import org.craftercms.studio.api.v2.repository.blob.StudioBlobStore;
 import org.craftercms.studio.api.v2.repository.blob.StudioBlobStoreResolver;
-import org.craftercms.studio.impl.v1.repository.git.GitContentRepository;
+import org.craftercms.studio.impl.v2.repository.GitContentRepository;
```

Comment:
Why? This breaks the system.

Reply:
Because below in the code V1 is declared with the full package name and V2 just class name (package is imported). Somewhere in history was a bad merge

Reply:
Did you fix the unit test to work with v2?

Reply:
yes